### DISCUSSION TITLE
Remix AppLoadContext should accept type via LoaderArgs

### DISCUSSION THREAD

**Discussion by 4x4notfound** [remix-run/remix#5173] [Q&A] [answered]
This would with type hints on the context shape

**Comment by sergiodxa** [comment, marked as answer]
If what you want is a way to overwrite AppLoadContext, you can do it from a .d.ts file like this https://github.com/sergiodxa/sergiodxa.com/blob/a3951ae2f439c8b15dfd777445052731f0b0eeb9/types/sdx.d.ts#L54

**Reply by 4x4notfound** [reply]
works :+1:
